tests/tel2puml/find_unique_graphs/test_otel_data_holder.py

```diff
@@ -0,0 +1,182 @@
+"""Tests for OTel data holder classes."""
+
+import pytest
+
+from datetime import datetime
+from unittest.mock import patch
+from sqlalchemy import text
+
+from tel2puml.find_unique_graphs.otel_ingestion.otel_data_model import (
+    OTelEvent,
+    NodeModel,
+    SQLDataHolderConfig,
+)
+from tel2puml.find_unique_graphs.otel_ingestion.otel_data_holder import (
+    SQLDataHolder,
+)
+
+
+class TestSQLDataHolder:
```

Comment:
Can we make sure that in tests for timestamps we are using probably accuracy of nano seconds just like the data we are collecting